cmd/podman/containers/create.go

```diff
@@ -189,6 +189,16 @@ func createInit(c *cobra.Command) error {
 		val := c.Flag("entrypoint").Value.String()
 		cliVals.Entrypoint = &val
 	}
+	groups, err := c.Flags().GetBool("groups")
+	if err != nil {
+		return err
+
+	}
+	if groups {
+		cliVals.Annotation = append(cliVals.Annotation, "run.oci.keep_original_groups=1")
+
+	}
+
```

Comment:
do we need to raise an error or at least a warning if the runtime is not crun?

Reply:
I did not want to do that right away, since crun and others might get this support.  But I can add a Warning.

Reply:
Concur, we need some sort of check that a compatible runtime is in use.